docs/resources/azapi_update_resource.md

```diff
@@ -129,6 +129,8 @@ The following arguments are supported:
 }
 ```
 
+* `locks` - (Optional) A list of ARM resource IDs which are used to avoid create/modify azapi resources at the same time.
```

Comment:
Hi @grayzu ,

This is used to handle case like, attach multiple associations to a resource but the API only allows attaching one association at a time. Then user can set `locks` to the resource ID, so multiple associations won't be created at the same time.

 Do you think the property name and its description make sense?

Reply:
Probably needs to mention it also affects the delete for the azapi_resource.md?